src/CCL.h

```diff
@@ -25,7 +26,7 @@ class CCL {
 				{ delete syms; syms = new_syms; }
 
 	unsigned int MemoryAllocation() const
-		{ return padded_sizeof(*this) + syms->MemoryAllocation(); }
+		{ return padded_sizeof(*this) + padded_sizeof(*syms); }
```

Comment:
Don't think this is equivalent since it's not counting the capacity allocated to the `std::vector` anymore ?

e.g. at a glance probably also need to add `pad_size(syms->capacity() * sizeof(ptr_compat_int))`

Reply:
This is definitely something that needs more testing. I'll take a look at it.

Reply:
You're correct here. `sizeof(vector)` only returns the allocated size of the vector itself, not the elements inside it. Fixed it in another fixup commit.